pkg/oci/utils_unix.go

```diff
@@ -53,6 +54,11 @@ func getDevices(path, containerPath string) ([]specs.LinuxDevice, error) {
 	if !stat.IsDir() {
 		dev, err := DeviceFromPath(path)
 		if err != nil {
+			// return error with detailed path and container path
+			if err == ErrNotADevice {
+				return nil, fmt.Errorf("get device path: %s containerPath: %s error: %w", path, containerPath, err)
```

Comment:
```suggestion
				return nil, fmt.Errorf("get device path: %q, containerPath: %q, error: %w", path, containerPath, err)
```

Reply:
@mxpv updated, thanks.

Now encounter CI issue, seems related to other.
https://github.com/containerd/containerd/actions/runs/8017117232/job/21900424559?pr=9858
>[FAILED] failed to pull image: rpc error: code = NotFound desc = failed to pull and unpack image "gcr.io/k8s-staging-cri-tools/win-test-image-tags:1": failed to prepare extraction snapshot "extract-860675400-qyfW sha256:5e0a6ea55afdc364445547977a75cae7b4fd9c603d3a595340865d8f541fa6c2": parent snapshot sha256:470cb2f9d9bb03a2065cf6cd75d764f68ea4a0de241105eb0cd3e90230be8163 does not exist: not found

Reply:
@mxpv When I changed the `%s` to `%q`, there are always CI failed instances; but when revert to `%s`, the CI does not fail. Do you know any reason? thanks.

Reply:
Judging from the above it might've been a red herring and a bit unlucky. Could you try %q one more time?

Reply:
@dcantah Re-committed, let's see if luck will come :), thanks.

Reply:
Looks all the CIs have passed.